conf/battle/player.conf

```diff
@@ -172,6 +172,10 @@ min_npc_vendchat_distance: 3
 // Default is 25. 100 = 100% Increase.
 rental_mount_speed_boost: 25
 
+// Display all status changes in the status window? (Note 1)
+// Default (official): no
+show_status_sc: no
```

Comment:
Do we want to have this config in `player.conf`? I think `status.conf` may suit better?